tests/cover/test_stateful.py

```diff
@@ -250,8 +250,7 @@ def test_bad_machines_fail(machine):
         raise
     v = o.getvalue()
     print_unicode(v)
-    assert u'Step #1' in v
-    assert u'Step #50' not in v
+    assert len(v.splitlines()) <= 50
```

Comment:
```python
assert 1 <= len(v.splitlines()) <= 50
```

Reply:
It needs a bit more of a change than that to still test the same thing (there might be other lines that aren't steps) but you're right this should be included. Have updated it to filter lines for ones that look like steps (which differs between the two types. I should probably update the output format for `GenericStateMachine` too but see comments about "I don't think anyone actually uses this..." for lack of motivation to do so)